extra/pattern/equilateral.js

```diff
@@ -0,0 +1,13 @@
+export default function equilateral(node){
+  let string = "";
+  for (let i = 1; i <= node; i++) {
```

Comment:
@arjavjainsuccessive, please mention the valid name instead of i, j, and string.